apps/src/netsim/NetSimDnsTable.html.ejs

```diff
@@ -0,0 +1,44 @@
+<%
+var NetSimRouterNode = require('./NetSimRouterNode');
+var DnsMode = NetSimRouterNode.DnsMode;
+%>
+<div class="netsim_dns_table">
+  <h1>My Network</h1>
+  <table>
+    <thead>
+    <tr>
+      <th>Hostname</th>
+      <th>Address</th>
+    </tr>
+    </thead>
+    <tbody>
+    <%
+    tableData.forEach(function (row) {
+      var displayHostname = row.hostname;
+      var displayAddress = '';
+      var rowClasses = [];
+
+      if (dnsMode === DnsMode.NONE || row.isDnsNode || row.isLocal) {
+        displayAddress = row.address;
+      }
+
+      if (row.isLocal) {
+        displayHostname += " (Me)";
```

Comment:
Not right now.  But often loc requires token substitution in case word order changes, rather than linear concatenation (e.g. +=) so just something to be aware of for down the road.